tests/foreman/cli/test_contentviewfilter.py

```diff
@@ -334,7 +334,7 @@ def test_create_cvf_repos_docker(self):
             self.assertIn(repo['id'], repos)
 
     @tier1
-    def test_create_cvf_names_negative(self):
+    def test_negative_create_with_name(self):
```

Comment:
`test_negative_create_with_invalid_name`? We used such approach for UI/API tests. I know you don't like it, `negative` and `invalid` share similar purpose, so it's not a blocker :)